PowerEditor/installer/nativeLang/bulgarian.xml

```diff
@@ -1045,9 +1045,9 @@
 		<MiscStrings>
 			<word-chars-list-tip value="Позволява добавяне на допълнителни символи към дума, чрез двойно щракане върху нея или при включена опция за търсене на &quot;Само цели думи&quot;."/>
 			<word-chars-list-warning-begin value="Бъдете внимателни: "/>
-				<!-- $INT_REPLACE$ is a place holder, don't translate it -->
+				<!-- $INT_REPLACE$ е контейнер, не се превежда -->
```

Comment:
It's really no need to translate it, just leave it.

Reply:
Corrected